modules/disconnected-osus-overview.adoc

```diff
@@ -11,4 +11,12 @@ The OpenShift Update Service (OSUS) provides update recommendations to {product-
 
 However, clusters in a disconnected environment cannot access these public APIs to retrieve update information. To have a similar update experience in a disconnected environment, you can install and configure the OpenShift Update Service locally so that it is available within the disconnected environment.
 
+A single OSUS instance is capable of serving recommendations to thousands of clusters.
+OSUS can be scaled horizontally to cater to more clusters by changing the replica value.
+So for most disconnected use cases, one OSUS instance is enough.
+For example, Red Hat hosts just one OSUS instance for the entire fleet of connected clusters.
+
+If for some reason you want to keep update recommendations separate in different environments, then you might want to run one OSUS instance for each environment.
+For example, a cluster in a stage environment should not get update recommendations to version A if the test environment has not updated to version A yet.
```

Comment:
If possible, I'd like a little more clarity about this use case so I could further refine this sentence.

Reply:
Example use-case:

1. Customer is running 4.y.z in their stage cluster pool and in their production cluster pool.
1. Red Hat GAs 4.y.z', adding it to the `fast-4.y` channel.
1. Customer mirrors 4.y.z' into their local registry with `oc-mirror`, which also updates the graph-data image in that registry.
1. Customer's stage OSUS picks up the new graph-data image, and begins recommending the update to 4.y.z' to the customer's stage clusters in the `fast-4.y` channel.  But at this point, the prod. OSUS is still not serving 4.y.z' in any channels.
1. Enough stage clusters update to 4.y.z' in the prod pool for the customer to feel confident with their workload's behavior on the new release, and the customer copies the graph-data image from its stage location to its production location.
1. Customer's prod. OSUS picks up the new graph-data image, and begins recommending the update to 4.y.z' to the customer's prod. clusters in the `fast-4.y` channel.

Customers doing that sort of thing would be running the two OSUS, one serving their stage pool, and another serving their production pool.  They would point those OSUS at different images for graph-data.  They might or might not run the two OSUS on the same host cluster.  They might or might not poitn the two OSUS at the same mirror registry for release images.

Reply:
Thank you for the clear explanation, I'll keep this in mind while potentially revising this line to be a little more clear to users about the use case.

I suppose one final question I have about this is the "why". Why would customers find value in making sure that the prod pool doesn't get recommended updates that they haven't yet tested? Are their potentially different groups of admins managing the different pools, and this is just one way to ensure that the prod admins don't accidentally update to an untested version?

Reply:
> ... and this is just one way to ensure that the prod admins don't accidentally update to an untested version?

Yeah, this.  I'm just imagining a hypothetical use-case; I'm not aware of anyone actually doing anything like this with OSUS.  But in this hypothetical world, there would be one customer team in charge of vetting updates/releases, and they would provide those options (once vetted)  to other customer teams in charge of triggering cluster updates from the set of options that the update/release-vetting team had approved (and was serving via a per-pool OSUS) for that particular cluster pool.